src/scss/components/news-vertical-teaser/_news-vertical-teaser.scss

```diff
@@ -118,45 +157,77 @@
     text-decoration: none;
   }
 
-  .su-card__link {
-    @include type_d;
-
-    color: $su-color-black;
-    text-decoration: none;
-
-    &:hover,
-    &:focus {
-      color: $chemh-color-digital-blue;
-      text-decoration: underline;
-    }
-
-    &:active {
-      color: $su-color-black;
-      text-decoration: underline;
-    }
-  }
-
   .news-vertical-teaser__footer {
+    margin: 0;
+
     .su-link--external-box {
       height: 50px;
       width: 50px;
 
       a {
-        span.material-symbols-outlined {
-          position: absolute;
-          right: 0;
-          bottom: 0;
-          padding: 1.3rem;
-          color: $su-color-white;
-          background-color: $chemh-color-dusk;
+        &.su-link--external {
+          padding-top: 1.3rem;
+
+          &::after {
+            content: "";
+            background: url("../../assets/svg/icon_arrow_up_right.svg") no-repeat 0 0;
```

Comment:
Would the link-icon mixin not work after all?

Reply:
I still don't understand why Font Awesome isn't being used for this situation. it's just an arrow. make it simple. you could remove many lines of css by just adding a `span` tag with the font awesome classes into the template. don't over complicate the development work just to match an icon.

Reply:
The icons I'm using here match the designs and match the icons we are already using on buttons. The available free FA icons didn't look good at all. It may save some css to use FA, but it will result in an uglier end-result 🤷

Reply:
We don't need to match the designs exactly. Designs are a starting point.  As a developer, you need to make choices that are best suited for the code. If there are slight variations, that is the result of a designer not using the tools we already have and not working with a developer to establish the designs.
"uglier"? explain how. The icons are nearly identical. Here's a screenshot of the two icons next to each other. The FA icon works. Literally all I did was added `<i class="fas fa-arrow-right" style="rotate: -45deg;"></i>` and it just worked. It is a simple solution and has the same results. Use the FA icons when available.

![image](https://github.com/SU-SWS/chem_h_subtheme/assets/7185045/65e15231-1cdd-4cf4-a8c8-4f8323c1a313)

Reply:
Updated to font-awesome link icons per related slack conversation